codebundles/k8s-tail-logs-dynamic/.runwhen/templates/k8s-tail-logs-dynamic-sli.yaml

```diff
@@ -0,0 +1,54 @@
+apiVersion: runwhen.com/v1
+kind: ServiceLevelIndicator
+metadata:
+  name: {{slx_name}}
+  labels:
+    {% include "common-labels.yaml" %}
+  annotations:
+    {% include "common-annotations.yaml" %}
+spec:
+  displayUnitsLong: OK
+  displayUnitsShort: ok
+  locations:
+    - {{default_location}}
+  description: Measures the health of a application workload by parsing for stacktraces in its logs.
+  codeBundle:
+    {% if repo_url %}
+    repoUrl: {{repo_url}}
+    {% else %}
+    repoUrl: https://github.com/runwhen-contrib/rw-cli-codecollection.git
+    {% endif %}
+    {% if ref %}
+    ref: {{ref}}
+    {% else %}
+    ref: main
+    {% endif %}
+    pathToRobot: codebundles/k8s-tail-logs-dynamic/sli.robot
+  intervalStrategy: intermezzo
+  intervalSeconds: 60
+  configProvided:
+  - name: KUBERNETES_DISTRIBUTION_BINARY
+    value: kubectl
+  - name: LOGS_SINCE
+    value: 10m
+  - name: LABELS
+    value: app={{match_resource.resource.metadata.labels.app}}
+  - name: EXCLUDE_PATTERN
+    value: Placeholder
```

Comment:
Is this a valid exclusion pattern? wouldn't it normally be blank in a template? or does the code handle this?

Reply:
Every setup has required this to be set in some way - INFO has been the most common pattern to include by default so I'm updating this to use that. The robot code does gracefully handle empty string.